### PR TITLE
[Doc] Add English Docs

### PR DESCRIPTION
**Pull request by michaelowenliu**

### CODE REVIEW

File: slim/quant/README.md [hunk 1]
```diff
@@ -1 +1 @@
-模型量化教程，请参考[文档](../../docs/slim/quant/quant.md)。
\ No newline at end of file
+Please refer to the [tutorial](../../docs/slim/quant/quant.md) for model quant.
```
Comment (juncaipeng) [inline slim/quant/README.md]: For model quantization ?

File: docs/slim/prune/prune.md [hunk 2]
```diff
@@ -64,26 +66,26 @@ python slim/prune/prune.py \
        --save_dir prune_model
 ```
 
-## 部署
+## Deployment
 
-通过`剪枝`得到的模型，我们可以直接进行部署应用，相关教程请参考[模型部署](../../model_export.md)。
+The pruned model can be deployed directly. Please refer to the [tutorial](../../model_export.md) for model deployment.
```
Comment (juncaipeng) [inline docs/slim/prune/prune.md:71]: This tutorial of exporting model is not English.

File: slim/quant/README.md (hunk 1 above)
Comment (michaelowenliu) [reply slim/quant/README.md]: done

File: docs/slim/prune/prune.md (hunk 2 above)
Comment (michaelowenliu) [reply docs/slim/prune/prune.md:71]: ?

Comment (juncaipeng) [reply docs/slim/prune/prune.md:71]: My fault.

**Review by juncaipeng — APPROVED** [review]
LGTM

**Pull request merged**